lib/assets/i18n/log/messages_en.json

```diff
@@ -229,51 +223,51 @@
   "ZWED0046W":"The auth services host system was not specified at startup, and defaulted to 127.0.0.1.\nVerify that the auth services server is running, or specify at startup the remote host and port to connect to. See documentation for details.",
   "ZWED0047W":"Swagger file for service (%s:%s) not found",
   "ZWED0048W":"Invalid Swagger from file for service (%s:%s)",
-  "ZWED0049W":"%s",
+  "ZWED0049W":"%s %s",
   "ZWED0050W":"Could not read swagger doc folder %s",
   "ZWED0051W":"Failed to parse translation file %s. File skipped",
   "ZWED0052W":"Error when reading file=%s. Error=%s",
-  "ZWED0053W":"Event handler failed: %s",
+  "ZWED0053W":"Event handler failed: ",
```

Comment:
The following few are same as above exclusion of arguments comment above

Reply:
normal